### PR TITLE
feat(cli): proxy-cookies flag to filter cookies

### PR DESCRIPTION
**Pull request by rafaelrubbioli**
fixes https://github.com/anycable/anycable-go/issues/156

### What is the purpose of this pull request?
Add a way to filter the cookies that are sent, to simplify and remove unneeded cookies

### What changes did you make? (overview)

I added a flag --proxy-cookies that, if set, will filter the cookies that are sent to the server

### Is there anything you'd like reviewers to focus on?

Please check if I did everything that was needed. I'd be happy to fix any problems and hear any suggestions 😄 

### Checklist

- [x] I've added tests for this change
- [ ] I've added a Changelog entry
- [x] I've updated documentation

Q: do i need to add a changelog entry?

### CODE REVIEW

**Comment by rafaelrubbioli** [comment]
@palkan Sure, will work on it!

**Comment by rafaelrubbioli** [comment]
@palkan Thanks for the tip! That was a great suggestion.

File: ws/handler.go [hunk 1]
```diff
@@ -66,7 +86,7 @@ func WebsocketHandler(headersToFetch []string, subprotocols []string, config *Co
 			url = fmt.Sprintf("%s%s%s", scheme, r.Host, url)
 		}
 
-		info, err := NewRequestInfo(r, headersToFetch)
+		info, err := NewRequestInfo(r, headersExtractor)
```
Comment (palkan) [inline ws/handler.go:89]: I think, we should pass `headersExtractor` here by reference (`&headersExtractor`) to avoid copying it for every request.

Comment (rafaelrubbioli) [reply ws/handler.go]: @palkan Done!

**Comment by rafaelrubbioli** [comment]
@palkan If everything is fine can you add the `hacktoberfest-accepted` label? Thanks!

**Comment by palkan** [comment]
Thanks!

Let's fix the linting errors and we're good to merge it.

**Comment by rafaelrubbioli** [comment]
> > there was an error on test-conformance, and I'm not really sure where to fix it
> 
> This is an integration test defined here: https://github.com/anycable/anyt/blob/ee8c622ff1b3c0431435a65e1047632c078208ba/lib/anyt/tests/request/connection_test.rb#L32
> 
> From the logs, we seen that no cookies are proxied (though we do not set `proxy-cookies`); looks like a bug.
> 
> UPD: See [go.dev/play/p/zToXcc6dBvP](https://go.dev/play/p/zToXcc6dBvP)

Good catch! Thats why we always have tests 😅 
I think it's fixed now

File: docs/configuration.md [hunk 2]
```diff
@@ -30,6 +30,10 @@ RPC service address (default: `"localhost:50051"`).
 
 Comma-separated list of headers to proxy to RPC (default: `"cookie"`).
 
+**--proxy-cookies** (`ANYCABLE_PROXY_COOKIES`)
+
+Comma-separated list of cookies to proxy to RPC (default: all cokies).
```
Comment (palkan) [inline docs/configuration.md:35]:
```suggestion
Comma-separated list of cookies to proxy to RPC (default: all cookies).
```

**Pull request merged**